neo/idlib/StrStatic.h

```diff
@@ -102,39 +102,39 @@ class idStrStatic : public idStr
 	{
 		buffer[ 0 ] = '\0';
 		SetStaticBuffer( buffer, _size_ );
-		idStr::operator=( b );
+		idStrStatic::operator=( b );
```

Comment:
maybe (probably) it was indented to call the baseclasses operator=()
(same of the other occurences)